xchange-okex/src/main/java/org/knowm/xchange/okex/v5/dto/marketdata/OkexPublicOrder.java

```diff
@@ -0,0 +1,75 @@
+package org.knowm.xchange.okex.v5.dto.marketdata;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.ObjectCodec;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+@JsonDeserialize(using = OkexPublicOrder.okexOrderDeserializer.class)
+public class OkexPublicOrder {
+
+  private final BigDecimal price;
+  private final BigDecimal volume;
+  private final Integer liquidatedOrders;
+  private final Integer activeOrders;
+
+  public OkexPublicOrder(
+      BigDecimal price, BigDecimal volume, Integer liquidatedOrders, Integer activeOrders) {
+
+    this.price = price;
+    this.volume = volume;
+    this.liquidatedOrders = liquidatedOrders;
+    this.activeOrders = activeOrders;
+  }
+
+  public BigDecimal getPrice() {
+
+    return price;
+  }
+
+  public BigDecimal getVolume() {
+
+    return volume;
+  }
+
+  @Override
+  public String toString() {
+    return "OkexPublicOrder{"
+        + "price="
+        + price
+        + ", volume="
+        + volume
+        + ", liquidatedOrders="
+        + liquidatedOrders
+        + ", activeOrders="
+        + activeOrders
+        + '}';
+  }
+
+  static class okexOrderDeserializer extends JsonDeserializer<OkexPublicOrder> {
```

Comment:
class names should start with large letters

BTW instead of writing a deserializer you can add these annotations

```
@JsonFormat(shape = JsonFormat.Shape.ARRAY)
@JsonPropertyOrder({"price", "volume", "liquidatedOrders", "activeOrders"})
public class OkexPublicOrder {
```

Reply:
Used annotations